### DISCUSSION TITLE
Große Tabellen - tbody scrollen

### DISCUSSION THREAD

**Discussion by webbird** [public-ui/kolibri#6989] [General]
Wenn man große - in diesem Fall vor allem lange - Tabellen auf einer Seite hat, die länger sind als der Viewport, ist es für Menschen ohne Beeinträchtigungen in der Regel angenehmer, wenn beim Scrollen der Tabellenheader stehen bleibt. Das läßt sich im einfachsten Fall nur mit CSS erreichen; für etwas mehr Komfort könnte man noch per JavaScript auf Veränderungen des Viewports reagieren oder ähnliches. Soweit ich das beurteilen kann, hat bisher kein Theme eine entsprechende Option. Mich würden die Meinungen der Community dazu interessieren bzw. ob es "in the wild" eventuell auch schon Themes gibt, die Tabellen entsprechend gestalten.

**Comment by webbird** [comment]
Siehe dazu auch folgende Diskussion: https://github.com/public-ui/kolibri/discussions/6976
Dort geht es zwar um das Umbrechen der Tabelleninhalte, wenn der Viewport zu schmal wird (=mobile Ansicht), aber die beiden Anforderungen sehe ich schon im Zusammenhang. Sie ergänzen sich so ein bißchen.

**Comment by deleonio** [comment]
Hey @webbird, wir haben in zwischen ein Sticky Header Beispiel umgesetzt, wo der Table-Body scrollbar ist: https://github.com/public-ui/kolibri/pull/8495
